valhalla/midgard/util.h

```diff
@@ -1,4 +1,5 @@
-#pragma once
```

Comment:
why change this? the project is slowly moving from old style header gaurds to the `pragma` style. i suppose i should just PR that to do it in one swoop

Reply:
Yeah, it should be all one or the other. I actually looked at other headers in this directory and thought this was a one-off, so I changed it to be consistent.